### ISSUE TITLE
node src/worker.js not working

### ISSUE THREAD

**Issue by burf2000** [spencermountain/dumpster-dive#33]
I am following you guide on how to do the redis way to speed stuff up, articles are loading in to redis but runnining node src/worker.js gives me lots of errors

TypeError: cb is not a function
    at Object.parse (/Users/snrb/Documents/wikipedia-to-mongodb-master/src/parse.js:65:12)
    at /Users/snrb/Documents/wikipedia-to-mongodb-master/src/worker.js:26:16
    at Worker.<anonymous> (/Users/snrb/Documents/wikipedia-to-mongodb-master/node_modules/kue/lib/queue/worker.js:230:7)
    at Job.<anonymous> (/Users/snrb/Documents/wikipedia-to-mongodb-master/node_modules/kue/lib/queue/job.js:706:12)
    at multi_callback (/Users/snrb/Documents/wikipedia-to-mongodb-master/node_modules/redis/lib/multi.js:89:14)
    at Command.callback (/Users/snrb/Documents/wikipedia-to-mongodb-master/node_modules/redis/lib/multi.js:116:9)
    at normal_reply (/Users/snrb/Documents/wikipedia-to-mongodb-master/node_modules/redis/index.js:721:21)
    at RedisClient.return_reply (/Users/snrb/Documents/wikipedia-to-mongodb-master/node_modules/redis/index.js:819:9)
    at JavascriptRedisParser.returnReply (/Users/snrb/Documents/wikipedia-to-mongodb-master/node_modules/redis/index.js:192:18)
    at JavascriptRedisParser.execute (/Users/snrb/Documents/wikipedia-to-mongodb-master/node_modules/redis-parser/lib/parser.js:574:12)

 at JavascriptRedisParser.execute (/Users/snrb/Documents/wikipedia-to-mongodb-master/node_modules/redis-parser/lib/parser.js:574:12)
TypeError: collection.insert is not a function
    at Object.parse (/Users/snrb/Documents/wikipedia-to-mongodb-master/src/parse.js:74:14)
    at /Users/snrb/Documents/wikipedia-to-mongodb-master/src/worker.js:26:16
    at Worker.<anonymous> (/Users/snrb/Documents/wikipedia-to-mongodb-master/node_modules/kue/lib/queue/worker.js:230:7)
    at Job.<anonymous> (/Users/snrb/Documents/wikipedia-to-mongodb-master/node_modules/kue/lib/queue/job.js:706:12)
    at multi_callback (/Users/snrb/Documents/wikipedia-to-mongodb-master/node_modules/redis/lib/multi.js:89:14)
    at Command.callback (/Users/snrb/Documents/wikipedia-to-mongodb-master/node_modules/redis/lib/multi.js:116:9)
    at normal_reply (/Users/snrb/Documents/wikipedia-to-mongodb-master/node_modules/redis/index.js:721:21)
    at RedisClient.return_reply (/Users/snrb/Documents/wikipedia-to-mongodb-master/node_modules/redis/index.js:819:9)
    at JavascriptRedisParser.returnReply (/Users/snrb/Documents/wikipedia-to-mongodb-master/node_modules/redis/index.js:192:18)
    at JavascriptRedisParser.execute (/Users/snrb/Documents/wikipedia-to-mongodb-master/node_modules/redis-parser/lib/parser.js:574:12)

**Comment by spencermountain** [comment]
hey Simon, sorry about this. This is just me being sloppy.
We've been working (hard!) on a new [re-write of this](https://github.com/spencermountain/wikipedia-to-mongodb/tree/dev) and i think a PR got merged into master maybe.

short story, we ran into a awful [process hanging](https://github.com/spencermountain/wikipedia-to-mongodb/issues/21) situation with one of our libraries, which brought us down the road to trying different ways of ingesting the dump. Fortunately, this dev branch version, though undocumented still, doesn't require redis stuff anymore, and is many-times faster.

I am working hard on getting this ready, but yeah, may have left master/redis-path in a broken state. I'm sorry bout this.

New version should be published next week, if everything goes well.
cheers
